service/controller/resource/tccpn/create.go

```diff
@@ -47,12 +47,6 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 			return nil
 		}
 
-		if cc.Status.TenantCluster.MasterInstance.EtcdVolumeSnapshotID == "" {
-			r.logger.LogCtx(ctx, "level", "debug", "message", "etcd volume snapshot id not available yet")
-			r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
-			return nil
-		}
-
```

Comment:
tccpn does not need to wait for that value anymore as the new cluster will be without `etcdsnapshot`

and for older cluster this  will be properly set as the snapshot is created and the resource that is fetching the id will be executed before this code